trace/trace.go

```diff
@@ -172,6 +210,68 @@ func (t *Trace) Send() {
 	}
 }
 
+// SpanContext contains information related to a Span. Context should be carried
+// around by the currently active span, and optionally propagated in trace context
+// headers.
+type SpanContext struct {
+	TraceID      string
+	ParentID     string
+	TraceContext map[string]interface{}
+	Dataset      string
+	TraceState   byte
+}
+
+// SerializeHeader returns a string representation, currently in Honeycomb trace
+// context header format.
+func (sc *SpanContext) SerializeHeader() string {
```

Comment:
ideally this wouldn't be exported from this package and would move to the honeycomb propagator instead.

Reply:
Yeah, that's much better. Thanks.